libraries/AP_Math/control.h

```diff
@@ -150,15 +150,21 @@ float kinematic_limit(Vector3f direction, float max_xy, float max_z_pos, float m
 // The expo should be less than 1.0 but limited to be less than 0.95.
 float input_expo(float input, float expo);
 
+// angle_rad_to_accel_mss converts a maximum lean angle in radians to an accel limit in m/s/s
+float angle_rad_to_accel_mss(float angle_rad);
+
 // angle_deg_to_accel_mss converts a maximum lean angle in degrees to an accel limit in m/s/s
 float angle_deg_to_accel_mss(float angle_deg);
 
+// accel_mss_to_angle_rad converts a maximum accel in m/s/s to a lean angle in radians
+float accel_mss_to_angle_rad(float accel_mss);
+
 // accel_mss_to_angle_deg converts a maximum accel in m/s/s to a lean angle in degrees
-float accel_mss_to_angle_deg(float accel);
-
-// rc_input_to_roll_pitch - transform pilot's normalised roll or pitch stick input into a roll and pitch euler angle command
-// roll_in_unit and pitch_in_unit - are normalised roll and pitch stick inputs
-// angle_max_deg - maximum lean angle from the z axis
-// angle_limit_deg - provides the ability to reduce the maximum output lean angle to less than angle_max_deg
-// returns roll and pitch euler angles in degrees
-void rc_input_to_roll_pitch(float roll_in_unit, float pitch_in_unit, float angle_max_deg, float angle_limit_deg, float &roll_out_deg, float &pitch_out_deg);
+float accel_mss_to_angle_deg(float accel_mss);
+
+// rc_input_to_roll_pitch_rad - transform pilot's normalised roll or pitch stick input into a roll and pitch euler angle command
+// roll_in_unit and pitch_in_unit - are normalised roll and pitch stick input
+// angle_max_rad - maximum lean angle from the z axis
+// angle_limit_rad - provides the ability to reduce the maximum output lean angle to less than angle_max_rad without changing the scaling
+// returns roll and pitch angle in radians
+void rc_input_to_roll_pitch_rad(float roll_in_unit, float pitch_in_unit, float angle_max_rad, float angle_limit_rad, float &roll_out_rad, float &pitch_out_rad);
```

Comment:
don't need to change this but we're using "norm" these days it seems instead of "unit"